clouddriver-kubernetes/clouddriver-kubernetes.gradle

```diff
@@ -61,6 +61,8 @@ dependencies {
   annotationProcessor "org.projectlombok:lombok"
   testAnnotationProcessor "org.projectlombok:lombok"
 
+  compile "io.kubernetes:client-java:11.0.1"
```

Comment:
Can we leave this as implementation?  [compile has been deprecated in gradle](https://docs.gradle.org/current/userguide/upgrading_version_5.html#deprecations) for a long time.

Reply:
Maybe the way forward for this PR is to add `testImplementation` ?

Reply:
I'm OK with `api` if we need it, but it doesn't seem like we do.  What's the error if there's no change to the dependencies?

Reply:
if we do not change the dependencies we got
```shell
> Task :clouddriver-web:compileTestJava FAILED
/Users/nemesis/armory/source/oss/clouddriver/clouddriver-web/src/test/java/com/netflix/spinnaker/clouddriver/config/ObjectMapperTest.java:24: error: package io.kubernetes.client.openapi.models does not exist
import io.kubernetes.client.openapi.models.V1Job;
                                          ^
/Users/nemesis/armory/source/oss/clouddriver/clouddriver-web/src/test/java/com/netflix/spinnaker/clouddriver/config/ObjectMapperTest.java:25: error: package io.kubernetes.client.openapi.models does not exist
import io.kubernetes.client.openapi.models.V1ObjectMeta;
                                          ^
/Users/nemesis/armory/source/oss/clouddriver/clouddriver-web/src/test/java/com/netflix/spinnaker/clouddriver/config/ObjectMapperTest.java:26: error: package io.kubernetes.client.openapi.models does not exist
import io.kubernetes.client.openapi.models.V1Pod;
                                          ^
/Users/nemesis/armory/source/oss/clouddriver/clouddriver-web/src/test/java/com/netflix/spinnaker/clouddriver/config/ObjectMapperTest.java:27: error: package io.kubernetes.client.openapi.models does not exist
import io.kubernetes.client.openapi.models.V1PodCondition;
                                          ^
/Users/nemesis/armory/source/oss/clouddriver/clouddriver-web/src/test/java/com/netflix/spinnaker/clouddriver/config/ObjectMapperTest.java:28: error: package io.kubernetes.client.openapi.models does not exist
import io.kubernetes.client.openapi.models.V1PodStatus;
                                          ^
/Users/nemesis/armory/source/oss/clouddriver/clouddriver-web/src/test/java/com/netflix/spinnaker/clouddriver/config/ObjectMapperTest.java:30: error: package org.joda.time does not exist
import org.joda.time.DateTime;
                    ^
```

Reply:
I'd use `testImplementation` to get around this.